src/client/Microsoft.Identity.Client/Http/HttpManager.cs

```diff
@@ -316,7 +277,7 @@ private async Task<HttpContent> CloneHttpContentAsync(HttpContent httpContent)
         /// In HttpManager, the retry policy is based on this simple condition.
         /// Avoid changing this, as it's breaking change.
```

Comment:
can we add a test to catch this if someone changes it?

Reply:
We surely have a quite a few of these.